spec/features/managing_settings_spec.rb

```diff
@@ -1,19 +1,51 @@
+require 'spec_helper'
```

Comment:
Do you all prefer single or double quotes here? I am seeing either or in the specs, bust mostly single quotes.